.env-example

```diff
@@ -4,4 +4,5 @@ APPLICATION_ID=YOUR_APPLICATION_ID
 PRIVATE_KEY=PATH_TO_YOUR_KEY_FILE
 FROM_NUMBER=YOUR_NEXMO_NUMBER
 TO_NUMBER=NUMBER_TO_CALL
-EVENT_URL=https://requestb.in/UUID_OR_https://UUID.ngrok.io
+CNAM=DO_INSIGHT_CNAM_LOOKUP
```

Comment:
could we change this to something that better implies the expected value here? Like TRUE,`TRUE_OR_FALSE`. Or adding a comment about accepted values?